### DISCUSSION TITLE
Does outlook calendar work with work/org account?

### DISCUSSION THREAD

**Discussion by servesh-chaturvedi** [add2cal/add-to-calendar-button#544] [Q&A] [answered]
Hi, thanks for this wonderful utility. I'm encountering a strange behavior where if I'm logged in using a work account and click on add to outlook, it is taking me to the sign in page. This issue is not happening with personal account. Can anyone please confirm this behavior if they have access to any work account. At this moment, I wasn't able to arrange for another work account but will keep the discussion updated. Thanks!

**Comment by jekuer** [comment, marked as answer]
For MS365 (usually what you mean by work account), you would need to go with the ms365 calendar type.
MS optimized it a little bit, but it is still some redirect hell.
The outlook.com option only works with private accounts (and the other way around).
You can change the calendar type labels to make it more clear to the user, if you want to.
Unfortunately, it is impossible from our side to detect which account the user is using.

**Reply by servesh-chaturvedi** [reply]
Thanks for the answer! Didn't know Microsoft has it this much complicated. I believe this info would be a great addition to the docs page itself.
